addons/web/static/src/views/graph/graph_model.js

```diff
@@ -443,6 +443,8 @@ export class GraphModel extends Model {
                                     }
                                     const num = numbers[id];
                                     label = num === 1 ? name : `${name} (${num})`;
+                                } else if (["many2many", "one2many"].includes(type)) {
```

Comment:
title message: [IMP] web: graph: group by a many2many

commit message:
- I would prefer many2many instead of m2m
- mention initial commit (or PR) that introduces the possibility to group by a many2many and precise that the first objective was to have that in gantt, pivot (and list?) --> your commit is not a fix